app/views/static_pages/index.html.erb

```diff
@@ -1 +1,80 @@
-<h1>chess-app - index.html.erb</h1>
\ No newline at end of file
+<h1>chess-app - index.html.erb</h1>
+
+<html>
+<body>
+  <div class="chessboard">
+    <!-- 1 -->
+    <div class="light"></div>
```

Comment:
Yes, definitely.  I will put it together along with the corrections and push it up today.